### ISSUE TITLE
MySQL 5.7 / 8.0.18-ol-7-r67: various DB_FLAVOR variable issues

### ISSUE THREAD

**Issue by cjohnweb** [bitnami/bitnami-docker-mysql#84]
**Description**
Previous to today this seemed to work fine.

Using docker-compose I was running bitnami/mysql5.7, upon running docker-compose up mysql would start and stop throwing the error that `/libmysql.sh: line 500: DB_FLAVOR: bad substitution`

I attempted to use a later version 8.0.18-ol-7-r67, which throws a similar error:
`/libmysql.sh: line 500: DB_FLAVOR: unbound variable`

**Steps to reproduce the issue:**

Create file docker-compose.yml (below) and run docker-compose up.

```
version: "3"
services:
  db:
    # image: "bitnami/mysql:5.7"
    image: "bitnami/mysql:8.0.18-ol-7-r67"
    restart: always
    container_name: db
    environment:
      - MYSQL_ROOT_PASSWORD="pass"
      - ALLOW_EMPTY_PASSWORD="no"
      - MYSQL_USER=myuser
      - MYSQL_PASSWORD="userpass"
      # - DB_FLAVOR="mysql"
    ports:
      - "3306:3306"
    networks:
      - db
    volumes:
      - /mysql-data:/bitnami/mysql
  phpmyadmin:
    image: phpmyadmin/phpmyadmin
    restart: always
    container_name: phpmyadmin
    ports:
      - 5050:80
    networks:
      - db
    depends_on:
      - db
    environment:
      # PMA_ARBITRARY: 1
      PMA_HOST: db
      PMA_PORT: 3306
networks:
  db:
    driver: bridge
```
**Describe the results you received:**
docker-compose attempts to restart mysql repeatedly, giving the same DB_FLAVOR errors each time.

**Describe the results you expected:**

MySQL would run like normal.

**Additional information you deem important (e.g. issue happens only occasionally):**

At first I was having issues authenticating with MySQL due to improper file permissions for my mounted volume. Once I fix that issue with a chmod, I was able to sign into phpmyadmin and successfully connect to MySQL. I switched over to write my mysql.js file to enable nodejs to connect to MySQL, and it would not connect. I switched over to the docker-compose screen and noticed the error. The error happens whether or not nodejs is running my server.

I have run `sudo docker system prune -a` multiple times to ensure I was pulling fresh containers.

**Version**
- Output of `docker version`:
```
Client:
 Version:           18.09.7
 API version:       1.39
 Go version:        go1.10.1
 Git commit:        2d0083d
 Built:             Fri Aug 16 14:20:06 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.09.7
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.1
  Git commit:       2d0083d
  Built:            Wed Aug 14 19:41:23 2019
  OS/Arch:          linux/amd64
  Experimental:     false
```
- Output of `docker info`:

```
Containers: 2
 Running: 0
 Paused: 0
 Stopped: 2
Images: 2
Server Version: 18.09.7
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 
runc version: N/A
init version: v0.18.0 (expected: fec3683b971d9c3ef73f284f176672c44b448662)
Security Options:
 apparmor
 seccomp
  Profile: default
Kernel Version: 4.15.0-66-generic
Operating System: Ubuntu 18.04.3 LTS
OSType: linux
Architecture: x86_64
CPUs: 2
Total Memory: 1.947GiB
Name: wou
ID: OYEP:O64X:EFO4:VJUC:BFBJ:7OAM:DNJJ:4TCW:RRFS:GGJZ:OON4:VWII
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Labels:
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false

WARNING: No swap limit support
```

- Output of `docker-compose version` (if applicable):

```
docker-compose version 1.17.1, build unknown
docker-py version: 2.5.1
CPython version: 2.7.15+
OpenSSL version: OpenSSL 1.1.1  11 Sep 2018
```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**
Running on Digital Ocean Ubuntu 18.04 LTS

**Comment by rafariossaa** [comment]
Hi @cjohnweb ,
  Thanks for noticing this.
  We will make a new release fixing this issue as soon as possible.
  We will come back to you.

**Comment by sameersbn** [comment]
@cjohnweb This issue has been resolved since `8.0.18-ol-7-r76`. Please reopen if issue persists
